docs/docs/get-started/quickstart.md

```diff
@@ -11,6 +11,7 @@ This is a quick Nebari CLI reference. If you're new to Nebari, start at [Install
 </div>
 
 ## Install
```

Comment:
@kenafoster, I know this is outside the scope of this PR, but could you also remove this installation section here? I think keeping this as a section here is redundant now, considering we have a whole page for that and we give a link to it at the beginning

Reply:
done